dev_sandbox/DataExploration_1D.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f00cf5bf",
+   "metadata": {},
+   "source": [
+    "# Milestone 1: Data Exploration\n",
+    "### Install Hints\n",
+    "Make sure to install your conda environment using the doc/envs/pydrop_dev.yml file for the most consistent results\n",
```

Comment:
suggestion(blocking): add a code block comment with the conda invocation to set up an env from that file.

Reply:
For this, could I refer to the README.md in the repo? I could say like "see ../README.md" for more info.